components/split-view/splitViewList.jsx

```diff
@@ -0,0 +1,276 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+/* eslint-disable class-methods-use-this */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { SPLIT_VIEW_LIST } from '../../utilities/constants';
+
+import Icon from '../icon';
+import SplitViewListItemContent from './private/list-item-content';
+import listItemWithContent from './private/list-item-with-Content';
+
+export const SORT_OPTIONS = Object.freeze({
+	UP: 'up',
+	DOWN: 'down'
+});
+
+const propTypes = {
+	/**
+	 * **Assistive text for accessibility**
+	 * * `list`: aria label for the list
+	 * * `sort`
+	 * * * `sortedBy`: Clickable sort header for the list.
+	 * * * `descending`: Descending sorting.
+	 * * * `ascending`: Ascending sorting.
+	 */
+	assistiveText: PropTypes.shape({
+		list: PropTypes.string,
+		sort: PropTypes.shape({
+			sortedBy: PropTypes.string,
+			descending: PropTypes.string,
+			ascending: PropTypes.string
+		}),
+		unreadItem: PropTypes.string
+	}),
+	/**
+	 * CSS classes to be added to the parent `div` tag.
+	 */
+	className: PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string]),
+	/**
+	 * Event Callbacks
+	 * * `onSelect`: Called when a list item is selected.
+	 * * * @param event {object} List item click event
```

Comment:
Not sure. React-docgen generates the docs json. 

What is the render engine?

Reply:
You can view the text that will goes into the doc site by running `npm run build-docs`. `components/component-docs.json` will be updated. The the doc site will read it as markdown.

I think all the `@param` and `{object}` need to be removed from the propType comments since they are used in the doc site. All "source code only" things like functions can keep the jsDoc stuff.